Sources/Rendering/Core/CellPicker/index.js

```diff
@@ -244,6 +254,88 @@ function vtkCellPicker(publicAPI, model) {
     return tMin;
   };
 
+  publicAPI.intersectVolumeWithLine = (p1, p2, t1, t2, tol, actor, mapper) => {
```

Comment:
maybe `actor, mapper` parameters could be grouped into a unique `volume` parameter instead.

Reply:
volumeActor is better? I can get the mapper form it

Reply:
In VTK's vocabulary, volume == `vtkImageData`, actor == `vtkPolyData`, so I think `volume` is enough here (and you get the mapper from it like you suggested).

Reply:
volume it is then...